src/scss/style.scss

```diff
@@ -1,4 +1,4 @@
-@charset 'utf-8';
+@charset "utf-8";
```

Comment:
was this change intentional?

Reply:
Must have been the IDE trying to be helpful